src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health) {
+    this.name = name;
+    this.health = 100;
```

Comment:
The `health` parameter is ignored in the Animal constructor, it always sets `this.health` to 100. If you want to set a default value if `health` is not passed, you can use `this.health = health || 100;`